alkira/data_source_alkira_oci_vcn_connector.go

```diff
@@ -0,0 +1,35 @@
+package alkira
```

Comment:
can we rename this file to data_source_alkira_connector_oci_vcn.go? So it matches our resource name convention.